src/languages/dotnet.md

```diff
@@ -0,0 +1,78 @@
+# .NET
+
+Platform.sh supports deploying .NET applications by allowing developers to define a build process and pass its variables to the .NET build environment.
+
+## Supported versions
+
+* 2.0
+* 2.1
+* 2.2
+
+To specify a .NET container, use the `type` property in your `.platform.app.yaml`.
+
+{% codesnippet "/registry/images/examples/full/dotnet.app.yaml", language="yaml" %}{% endcodesnippet %}
+
+## Building the application
+
+.NET containers do not come with a pre-defined build process that you may have seen on other container types. Instead, the process is defined by you, using the runtime and output directory variables passed to the build environment.
```

Comment:
I'm not sure this is relevant.  Only two container types, PHP and node, include "flavors" as we've been moving away them.  We don't have this comment on any other language AFAIK.

Reply:
Removing.